### PR TITLE
Install new nuget for protos. Update references.

### LINKED ISSUE #261: Create nuget repo for C# protobuf contracts
Currently the protobuf files used by the campaign module are linked and generated directly in the repo itself. 
The protobuf files have recently been also moved to a new [repository](https://github.com/daritelska-platforma/types/).

A nuget repository can be created for the organization where the protobuf contracts can be uploaded and maintained separately from everything else. C# modules/microservices can then use that repo to pull the protobuf library.

### PR DESCRIPTION
**Pull request by preslavgerchev**
Uncoupled the Campaigns module from the proto files and installed the new nuget package.

Closes #261

### CODE REVIEW

File: backend/module-campaigns/nuget.config [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="podkrepibg-github" value="https://nuget.pkg.github.com/podkrepi-bg/index.json" />
+  </packageSources>
+  <packageSourceCredentials>
+	<podkrepibg-github>
+		<add key="Username" value="devops-podkrepi-bg" />
+		<add key="ClearTextPassword" value="&#103;hp_wZMwRA1oE6hwdsYDCziboiZFpOkkZs0eaaAt" />
```
Comment (kachar) [inline backend/module-campaigns/nuget.config:9]: I'm not aware of the scope but is this a secure way of using the password?

Comment (imilchev) [reply backend/module-campaigns/nuget.config:9]: NuGet packages in GitHub cannot be public. The token that is added to the repo has read access for packages so it should be okay. Other people are also working around the private NuGet packages in this way.

Comment (kachar) [reply backend/module-campaigns/nuget.config:9]: Thanks for bringing up that knowledge :books:

**Comment by kachar** [comment]
@xenry @mayapeneva Can you please check this PR and merge it whenever you're available if nothing else needs to be added

**Review by mayapeneva — APPROVED** [review]

File: backend/module-campaigns/src/Podkrepibg.Campaigns.Application/Podkrepibg.Campaigns.Application.csproj [hunk 2]
```diff
@@ -16,12 +16,8 @@
     <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.5" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="5.0.0" />
-    <PackageReference Include="Grpc.AspNetCore" Version="2.37.0" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Protobuf Include="..\..\..\protos\campaign.proto" GrpcServices="Server" ProtoRoot="..\..\..\protos\" />
-    <Protobuf Include="..\..\..\protos\common\nomenclatures.proto" GrpcServices="Server" ProtoRoot="..\..\..\protos\" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.38.0" />
```
Comment (xenry) [inline backend/module-campaigns/src/Podkrepibg.Campaigns.Application/Podkrepibg.Campaigns.Application.csproj:19]: maybe we can remove this nuget from here as it is no longer needed? We can put it in the main project (Podkrepibg.Campaigns). If we do so, we have to add a reference to `Microsoft.Extensions.Configuration.Abstractions` package here.

**Review by xenry — APPROVED** [review]

**Pull request merged**